web/public/js/directives/cd-add-child/template.dust

```diff
@@ -0,0 +1,70 @@
+      <div class="input-group">
+        <input type="text" class="form-control" readonly uib-datepicker-popup="dd-MMMM-yyyy" datepicker-options="$ctrl.dobDateOptions" popup-placement="bottom-left" datepicker-append-to-body="true" min-date="'1900-01-01'" max-date="today" ng-model="$ctrl.profile.dob" name="cdAddChildDob" required-message="'{@i18n key="Date of birth is empty"/}'" is-open="$ctrl.dobIsOpen" required current-text="{@i18n key="Today"/}" clear-text="{@i18n key="Clear"/}" close-text="{@i18n key="Done"/}" placeholder="{@i18n key="Date of birth"/}" ng-click="$ctrl.dobIsOpen = !$ctrl.dobIsOpen" />
+        <span class="input-group-btn" ng-class="{'topValign':cdAddChildForm.$submitted && cdAddChildForm.cdAddChildDob.$error.required}">
+          <button type="button" class="btn btn-default" ng-click="$ctrl.dobIsOpen = !$ctrl.dobIsOpen"><i class="glyphicon glyphicon-calendar"></i></button>
+        </span>
+      </div>
+      <span class="help-block">{@i18n key="We need your child's date of birth to comply with child protection laws in some countries."/}</span>
+    </div>
+  </div>
+  <div class="form-group form-inline">
+    <label for="cd-add-child__gender" class="col-sm-2 control-label">Gender</label>
+    <div class="col-sm-10">
+      <div class="radio">
+        <input type="radio" id="cd-add-child__gender-male" name="cd-add-child__gender" value="Male" ng-model="$ctrl.profile.gender" />
+        <label for="cd-add-child__gender-male">Male</label>
+      </div>
+      <div class="radio">
+        <input type="radio" id="cd-add-child__gender-female" name="cd-add-child__gender" value="Female" ng-model="$ctrl.profile.gender" />
+        <label for="cd-add-child__gender-female">Female</label>
+      </div>
+      <div class="radio">
+        <input type="radio" id="cd-add-child__gender-undisclosed" name="cd-add-child__gender" value="Undisclosed" ng-model="$ctrl.profile.gender" />
+        <label for="cd-add-child__gender-undisclosed">Undisclosed</label>
+      </div>
+    </div>
+  </div>
+  <div class="cd-add-child__o13-section" ng-show="$ctrl.profile.dob < $ctrl.thirteenYearsAgo">
+    <div class="col-sm-10 col-sm-offset-2 cd-add-child__info-box">
+      <p>{@i18n key="As your child is over 13 years of age, they can have their own CoderDojo account."/}</p>
+      <p>{@i18n key="This is optional, but will allow your child to log in by themselves to access features such as badging, booking and the youth forums by themselves. Enter your child's email address, and give them a password if you wish to do this."/}</p>
+    </div>
+    <div class="form-group">
+      <label for="cd-add-child__email" class="col-sm-2 control-label">Email</label>
+      <div class="col-sm-10">
+        <input type="email" class="form-control" name="cd-add-child__email" placeholder="Email" ng-model="$ctrl.profile.email" validator="$ctrl.emailValidator()" invalid-message="'{@i18n key="You must provide an email address when providing a password."/}'">
```

Comment:
You can use this if the validator is supposed to be reusable https://docs.angularjs.org/api/ng/type/ngModel.NgModelController#$validators